applications/picocdn-node-vm/templates/_helpers.tpl

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 NetPico LAbs Pvt. Ltd.(picoNETS)
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "picocdn-node-vm.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "picocdn-nocd-vm.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
```

Comment:
Rephrase: (THESE ARE COSMETIC CHNAGES - SEE IF POSSIBLE TO RESOLVE THESE COMMENTS)
The app name is truncated to 63 characters as some of the Kuberenetes name fields are limited ti this (by the DNS naming spec).

Reply:
Re-written the comment in a neutral perspective, should now avoid the first/second person grammatical constructs.